### PR TITLE
fix wrong chain metamask alert issue

### PR DESCRIPTION
**Pull request by topdevgh**

### CODE REVIEW

File: .vscode/settings.json
Comment (wolfwarrior14) [inline .vscode/settings.json:1]: Please remove this thing.

Comment (wolfwarrior14) [inline .vscode/settings.json:1]: Please don't make changes in project settings.

**Review by wolfwarrior14 — COMMENTED** [review]
Looks good

File: .vscode/settings.json
Comment (topdevgh) [reply .vscode/settings.json:1]: fixed

**Review by wolfwarrior14 — DISMISSED** [review]

**Review by qudacki — APPROVED** [review]

**Pull request merged**